include/z64player.h

```diff
@@ -598,12 +602,12 @@ typedef struct Player {
     /* 0xADE */ u8 unk_ADE;
     /* 0xADF */ s8 unk_ADF[4];
     /* 0xAE3 */ s8 unk_AE3[4];
-    /* 0xAE7 */ s8 unk_AE7;  // a timer
+    /* 0xAE7 */ s8 unk_AE7; // a timer
     /* 0xAE8 */ s16 unk_AE8; // multipurpose timer
     /* 0xAEC */ f32 unk_AEC;
     /* 0xAF0 */ Vec3f unk_AF0[2];
     /* 0xB08 */ f32 unk_B08[8];
-    /* 0xB28 */ s16 unk_B28; // Burning stick timer?
+    /* 0xB28 */ s16 unk_B28; //Burning stick timer?
```

Comment:
```suggestion
    /* 0xB28 */ s16 unk_B28; // Burning stick timer?
```

Reply:
Left this one out, as the PR no longer touches z64player.h